app/src/views/GrantRoundGrants.vue

```diff
@@ -1,9 +1,8 @@
 <template>
-  <template v-if="hasLoadedHeader">
+  <template v-if="hasLoadedHeader && hasLoadedGrants">
```

Comment:
Why did we move `hasLoadedGrants` from the `GrantList` component to the `template` element?

Reply:
You're right, there is no need. Just put it back https://github.com/dcgtc/dgrants/pull/277/commits/0cad7951dcdf9f12c4d569e5cd898228da7ead1d